Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -189,5 +199,11 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         bodyField.becomeFirstResponder()
         return false
     }
+    
+    /// Called when editing changed on the title field, enable/disable submit button based on title text
+    @objc
+    func textFieldEditingChanged(_ textField: UITextField) {
```

Comment:
Since this controller is currently in a Storyboard, would it be better to hook up the `.editingChanged` event as an `@IBAction`? It doesn't make a difference functionally but it is less code.

I see it [used](https://github.com/rnystrom/GitHawk/search?utf8=✓&q=ibaction&type=) in a few places in the codebase.

Reply:
@Sherlouk `IBAction` or target which one you prefer? here

Reply:
If we can get away with `IBAction` then sure, cleans it self up better 😃

Reply:
Done! lot simpler now with `IBAction`